react/features/remote-video-menu/components/MuteButton.js

```diff
@@ -96,6 +97,14 @@ class MuteButton extends Component {
     _onClick() {
         const { dispatch, onClick, participantID } = this.props;
 
+        JitsiMeetJS.analytics.sendEvent(
+            'remotevideomenu.mute',
+            {
+                value: 1,
```

Comment:
Why value 1?

Reply:
I was thinking it may be useful to count the total mutes in a call. I'm not sure if this will be possible if I don't pass a value.

Reply:
So a value has to exist in order to do a count on the recorded remotevideomenu.mute events? The event itself can't be counted otherwise? Out of curiosity can value then be any arbitrary value?

Reply:
I'm not sure if the event can't be counted otherwise.

The value for Google analytics must be number I think. Probably for other analytics providers you can put whatever you want.